core/build.gradle.kts

```diff
@@ -84,4 +84,7 @@ dependencies {
   implementation(Libs.webkit)
   testImplementation(Libs.kotlinx_coroutines_test)
   implementation(Libs.kotlinx_coroutines_android)
+
+  // ZXing
```

Comment:
Please remove this comment as the `implementation(Libs.zxing)` already shows this is a Zxing library.

Reply:
🚮 Removed